app/src/main/java/com/goldenraven/padawanwallet/ui/wallet/WalletViewModel.kt

```diff
@@ -48,30 +46,35 @@ private const val TAG = "WalletViewModel"
 class WalletViewModel(
     application: Application,
 ) : AndroidViewModel(application) {
-    val readAllData: LiveData<List<Tx>>
+    val readAllData: MutableStateFlow<List<Tx>> = MutableStateFlow<List<Tx>>(emptyList<Tx>())
     private val repository: TxRepository
     var openFaucetDialog: MutableState<Boolean> = mutableStateOf(false)
 
     private var _balance: MutableStateFlow<ULong> = MutableStateFlow(0u)
     val balance: StateFlow<ULong>
         get() = _balance
 
-    private var _address: MutableLiveData<String> = MutableLiveData("")
-    val address: LiveData<String>
+    private var _address: MutableStateFlow<String> = MutableStateFlow("No address yet")
```

Comment:
No need to fix it in this PR but we should address this initial address state plus the ideas of maybe using a progress bar or some visual as it's being generated described in #294. For now this is what the ReceiveScreen looks like (not very intuitive in my opinion):
<img width="352" alt="address" src="https://github.com/thunderbiscuit/padawan-wallet/assets/39974688/40a94fd3-a506-487a-8858-efcc8093ea37">

Reply:
Fixed with #319.